plugins/spammer/cpu_usage.go

```diff
@@ -0,0 +1,107 @@
+package spammer
+
+import (
+	"errors"
+	"io/ioutil"
+	"math/rand"
+	"runtime"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+)
+
+var (
+	CPUUsageTimePerSample = time.Second / 4
+
+	activeWorkerCount = 0
+
+	once sync.Once
+	mu   sync.Mutex
+
+	cpu_last_sum uint64   // previous iteration
+	cpu_last     []uint64 // previous iteration
+
+	cpuUsageResult float64 // current result
+	cpuUsageError  error   // nil || current error
+)
+
+func cpuUsageUpdater() {
+	if runtime.GOOS == "windows" {
+		mu.Lock()
+		defer mu.Unlock()
+		cpuUsageError = errors.New("spammer.maxCPUUsage on Windows is not supported")
+		return
+	}
+
+	go func() {
+		for {
+			procStat, err := ioutil.ReadFile("/proc/stat")
```

Comment:
Don't use that reading of `/proc/stat`.
We already use the `psutil` package somewhere else. Just use the `Percent()` function in that package.